### ISSUE TITLE
Additional step for Diff Exp tutorial

### ISSUE THREAD

**Issue by AlexanderPico** [cytoscape/cytoscape-tutorials#50]
[A recent helpdesk question](https://groups.google.com/g/cytoscape-helpdesk/c/LA6s1S7fPg4/m/eHONlk3EAAAJ
) revealed that we are missing a major step in our[ intro tutorial on diff exp](https://cytoscape.org/cytoscape-tutorials/protocols/differentially-expressed-genes/). We start with upregulated gene lists... but we don't have any fold change data associated with them. This is an odd case.  We only show data overlay for the generic disease case and miss showing how to map your own data to your own protein query network.

Things to change:

- We should prepare sample dataset with gene names, fold change and adjusted pvalues. 
- STRING query would be the same (using the gene names) https://cytoscape.org/cytoscape-tutorials/protocols/differentially-expressed-genes/#/5
- The next step should be to upload the data and perform a basic node fill color mapping. This should happen before the enrichment analysis.
- Skip the down regulated case. Just do a single example to keep it simple. The example could be BOTH up and down or just one or the other.

**Comment by AlexanderPico** [comment]
Even better, we should extract out this simple, most common use case on it's own:

1. query STRING by gene name (based on example dataset)
2. load data (mapping via "query term" column)
3. style network
4. comment on largest connected subnetwork

This should be separated from the disease query use case.  So, maybe just split the current tutorial into two: diff exp network viz and disease network viz.

**Comment by khanspers** [comment]
Update: 
> We should prepare sample dataset with gene names, fold change and adjusted pvalues.

=> done, except we are using gene identifier
> STRING query would be the same (using the gene names) https://cytoscape.org/cytoscape-tutorials/protocols/differentially-expressed-genes/#/5
> The next step should be to upload the data and perform a basic node fill color mapping. This should happen before the enrichment analysis.

=> done

> Skip the down regulated case. Just do a single example to keep it simple. The example could be BOTH up and down or just one or the other.

=> up/down is one example, disease is another (in the same protocol)

As for the proposed simplified workflow (query STRING by gene name (based on example dataset); load data (mapping via "query term" column); style network; comment on largest connected subnetwork), we don't have this protocol yet. Will open a separate issue for that.
